docs/content/providers/docker.md

```diff
@@ -721,8 +721,7 @@ providers:
     consider the [Enterprise Edition](https://traefik.io/traefik-enterprise/).
     You can use it as your:
 
-    - [Kubernetes Ingress Controller](https://traefik.io/solutions/kubernetes-ingress/)
-    - [Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/)
+    - [Docker Router and Load Balancer](https://traefik.io/solutions/docker-swarm-ingress/)
```

Comment:
why do we have this part "customized" in the docker provider, and not in the other providers?

for example, why not in docs/content/providers/kubernetes-crd.md removing the bullet about https://traefik.io/solutions/docker-swarm-ingress/ ?

what's the rationale/logic here?

Reply:
I tried to only provide relevant links where I could. So in this case, because it's docker, it doesn't make sense to have the K8s info. That said, I may have overlooked another one that should have been customized.

We are updating the docker swarm load balancer paid to be all about load balancers, not just for docker swarm, so your latter example will be relevant pretty soon.